setup.py

```diff
@@ -41,6 +41,7 @@
 extras = {}
 extras["testing"] = ["pytest", "pytest-mock", "pytest-cov", "pytest-xdist"]
 extras["dev"] = extras["testing"] + ["colorama==0.4.3"]
+extras["j2k"] = ["pynvjpeg2k @ git+ssh://git@github.com/medcognetics/pynvjpeg2k.git"]
```

Comment:
Can we updated this message to reflect the `j2k` extra?
https://github.com/medcognetics/dicom-utils/blob/master/dicom_utils/dicom.py#L406